src/wattsi.pas

```diff
@@ -1999,11 +2007,25 @@ procedure Inform(Message: UTF8String);
       Writeln(Message);
 end;
 
+// http://wiki.freepascal.org/UTF8_strings_and_characters#Search_and_copy
+function SplitInHalf(Txt, Separator: UTF8String; out Half1, Half2: UTF8String): Boolean;
+var
+  i: Integer;
+begin
+  i := Pos(Separator, Txt);
```

Comment:
Just use whatever works. :-) Maybe drop a link to this review so that someone in the future can improve it if it turns out to be a bottleneck.